lib/diagnostic.rb

```diff
@@ -115,10 +119,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {age: 42, name: Douglas}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.default = `#{:first_name}+ 'brought his towel'`'
```

Comment:
You have to call the object first. so it would be:
```ruby
Diagnostic.hash_default = 'person.default = "#{person[:first_name]} 'brought his towel'
```